applications/prompt-proto-service-hsc-gpu/README.md

```diff
@@ -36,13 +36,13 @@ Prompt Proto Service is an event driven service for processing camera images. Th
 | prompt-proto-service.knative.cpuRequest | int | `1` | The cpu cores requested. |
 | prompt-proto-service.knative.ephemeralStorageLimit | string | `"5Gi"` | The maximum storage space allowed for each container (mostly local Butler). |
 | prompt-proto-service.knative.ephemeralStorageRequest | string | `"5Gi"` | The storage space reserved for each container (mostly local Butler). |
+| prompt-proto-service.knative.extraTimeout | int | `10` | To acommodate scheduling problems, Knative waits for a request for twice `worker.timeout`. This parameter adds extra time to that minimum (seconds). |
 | prompt-proto-service.knative.gpu | bool | `true` | GPUs enabled. |
 | prompt-proto-service.knative.gpuRequest | int | `1` | The number of GPUs to request. |
-| prompt-proto-service.knative.idleTimeout | int | `900` | Maximum time that a container can send nothing to the fanout service (seconds). |
+| prompt-proto-service.knative.idleTimeout | int | `0` | Maximum time that a container can send nothing to the fanout service (seconds). This is only useful for async workers. If 0, idle timeout is ignored. |
```

Comment:
What do you mean by async workers?  Knative does not support async.

Reply:
I was referring to [Gunicorn](https://docs.gunicorn.org/en/stable/design.html) support for intermediate responses and keepalives. I think in addition to switching worker type we'd have to use something other than Flask for the actual request processing?

Reply:
Oh I see.  Thanks.  Correct. 

Something like this?  "Knative timeout values.  Note that gunicorn/flask timeout values are set"